auto_submit/lib/service/config.dart

```diff
@@ -48,6 +48,7 @@ class Config {
   /// Labels autosubmit looks for on pull requests
   static const String kAutosubmitLabel = 'autosubmit';
   static const String kRevertLabel = 'revert';
+  static const String kRevertOfLabel = 'revert of';
```

Comment:
How these two labels `revert` and `revert of` will be used?

Reply:
Ah good question:

Labels:
'revert' tells the service that this is a closed and merged change that will need to be reverted.
'revert of' tells the service this is a new pull request that reverts an already merged change. This pull request is open and unmerged.

Reply:
I unfortunately had to do this because of the events generated. Using two labels prevents me from having to compare a bunch of other fields. This is much more robust.

Reply:
How these two labels will be added to the PR, namely how is the workflow? Do people need to add these two labels to both PRs, or they just need to add `revert` to the to-be-reverted PR, and then the auto submit bot creates the revert PR and auto adds the `revert of` label?

Reply:
Based on offline discussion. It is the latter case. `revert of` label will be handled/used by the auto submit bot only, and people only need to add `revert` label to the to-be-reverted PR.

Nit: @ricardoamador Could you help add a brief doc for these two labels?

Reply:
Yes sir!

Reply:
Added a comment with an issue to link the single page design doc when done.